styles/global.css

```diff
@@ -11,13 +11,6 @@ html {
   font-weight: 400;
   src: url(/fonts/DM-mono-v10.ttf) format('truetype');
```

Comment:
why only the sans font, but not the mono?

also: what is this `DM Serif Display`?

Reply:
Because they are in use right now. We could change them later but it's not the goal of this PR>